File: Srims.WebSite/javascript/srims/CensorState.js
Srims.CensorState = new function(){
};

Srims.CensorState.unSubmited='UnSubmited';
Srims.CensorState.waitingCensor='WaitingCensor';
Srims.CensorState.reject='Reject';
Srims.CensorState.passed='Passed';
Srims.CensorState.canceled='Canceled';

Srims.CensorState.Render = function(value){
    switch (value) {
        case 'UnSubmited':
            return '未提交';
        case 'WaitingCensor':
            return '等待审核';
        case 'Reject':
            return '审核驳回';
        case 'Passed':
            return '审核通过';
        case 'Canceled':
            return '作废';
        default:
            return '<span class="error">未知</span>';
    }
};

File: Srims.Server.Business/CensorState.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace Srims.Server.Business
{
    /// <summary>
    /// 审核状态
    /// </summary>
    public enum CensorState
    {
        /// <summary>
        /// 未知状态
        /// </summary>
        Unknown = 0,
        /// <summary>
        /// 未提交
        /// </summary>
        UnSubmited = 1,
        /// <summary>
        /// 等待审核
        /// </summary>
        WaitingCensor = 2,
        /// <summary>
        /// 驳回
        /// </summary>
        Reject = 3,
        /// <summary>
        /// 审核通过
        /// </summary>
        Passed = 4,
        /// <summary>
        /// 已作废
        /// </summary>
        Canceled = 5
    }
}
